libs/portals/admin/ids-admin/src/shared/hooks/useCopyToClipboard.ts

```diff
@@ -0,0 +1,23 @@
+import { RefObject, useCallback } from 'react'
+
+import { toast } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+
+import { m } from '../../lib/messages'
+
+export const useCopyToClipboard = () => {
```

Comment:
Nice! Very useful hook. I am wondering if it would be more explicit usage of the function to name it `copyToClipboard` instead of `handleCopy`